DigitalOffice/Tests/CompanyServiceTests/Repositories/CompanyRepositoryTests.cs

```diff
@@ -0,0 +1,57 @@
+﻿using LT.DigitalOffice.CompanyService.Database;
+using LT.DigitalOffice.CompanyService.Repositories;
+using LT.DigitalOffice.CompanyService.Database.Entities;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using LT.DigitalOffice.CompanyServiceUnitTests.UnitTestLibrary;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Repositories
+{
+    public class CompanyRepositoryTests
+    {
+        private CompanyServiceDbContext dbContext;
+        private ICompanyRepository repository;
+
+        private DbCompany dbCompanyInDB;
+
+        [OneTimeSetUp]
+        public void OneTimeSetUp()
+        {
+            var dbOptions = new DbContextOptionsBuilder<CompanyServiceDbContext>()
+                                    .UseInMemoryDatabase(databaseName: "InMemoryDatabase")
+                                    .Options;
+            dbContext = new CompanyServiceDbContext(dbOptions);
+            repository = new CompanyRepository(dbContext);
+
+            dbCompanyInDB = new DbCompany
+            {
+                Id = Guid.NewGuid(),
+                Name = "Lanit-Tercom",
+                IsActive = true
+            };
+
+            dbContext.Companies.Add(dbCompanyInDB);
+            dbContext.SaveChanges();
+        }
+
+        [TearDown]
+        public void Clear()
+        {
+            if (dbContext.Database.IsInMemory())
+            {
+                dbContext.Database.EnsureDeleted();
+            }
+        }
+
```

Comment:
Add test that checks throwing exception if there are no such company in db.

Reply:
Fixed.